pkgs/os-specific/linux/kernel/hardened/update.py

```diff
@@ -145,7 +145,7 @@ def find_asset(filename: str) -> str:
     if not sig_ok:
         return None
 
-    kernel_ver = re.sub(r"(.*)(-hardened[\d]+)$", r'\1', release_info.release.tag_name)
+    kernel_ver = re.sub(r"v?(.*)(-hardened[\d]+)$", r'\1', release_info.release.tag_name)
```

Comment:
fwiw is this even necessary? There's no `^` at the start of the regex.

Reply:
It is necessary, due to the following capture group `(.*)` which then gets outputted `\1`, that is to drop any initial v if present